extensions/sld/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2020, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+set(EXT_NAME ext-sld)
+
+set(EXT_SOURCES
+    src/BusbarSectionPosition.cpp
+    src/BusbarSectionPositionAdder.cpp
+    src/BusbarSectionPositionXmlSerializer.cpp
+    src/Sld.cpp
```

Comment:
I wonder if we should rename this file sld.cpp as it doesn't contain any types. It's done like this for other modules, so maybe we can do this in another PR.